recipe/meta.yaml

```diff
@@ -19,6 +18,11 @@ requirements:
     - pip
   run:
     - python
+    - async-timeout >=4.0.2
+    - deprecated >=1.2.3
+    - importlib-metadata >=1.0
```

Comment:
Should be `importlib-metadata >=1.0  # [py<38]`, see https://github.com/redis/redis-py/blob/e6cd4fdf3b159d7f118f154e28e884069da89d7e/setup.py#L37